### DISCUSSION TITLE
step by step wizard edit mode

### DISCUSSION THREAD

**Discussion by Nivetha2510** [serenity-is/Serenity#6368] [Help]
Hi
In serenity pro,Is there a way to open a wizard form in edit mode with customer details.I need a wizard form in edit mode too.Thanks for any support in advance

**Comment by VictorTomaili** [comment]
Use serenity pro repository for pro related questions.

**Reply by Nivetha2510** [reply]
can u share that repository

**Reply by VictorTomaili** [reply]
You should already access the repository if you have a pro subscriiption. 
If you wrote the wrong GitHub account name while you buy the serenity pro, you can change and update it by sending an email to the sales which are provided on `serenity.is` website under the package list.

**Reply by Nivetha2510** [reply]
ok.Thanks
